### DISCUSSION TITLE
Need to fix russian translation on power configuration pannel

### DISCUSSION THREAD

**Discussion by hexesisb** [elementary/switchboard-plug-power#234] [Q&A] [answered]
Show accumulator % in PANEL - there is a mistake, should be:
Показывать заряд аккумулятора на **панели**
![image](https://github.com/elementary/switchboard/assets/134764601/8f14d359-2d25-40c6-bc4e-87ef0e108d4b)

**Comment by ryonakano** [comment, marked as answer]
Hi, thanks for pointing out this!

We're managing translations using a web hosting service called Weblate, and it looks like this translation has been already changed to what you suggested on Weblate:

https://l10n.elementary.io/translate/switchboard/switchboard-plug-power/ru/?checksum=4f48d4076972280d

However, the reason why the old translation remains is just because a new version of powe plug not yet released, so closing this issue as resolved.
